templates/openstack/install.html

```diff
@@ -41,56 +41,52 @@ <h1>Install OpenStack yourself</h1>
 <section class="p-strip--light is-bordered">
   <div class="row">
     <div class="col-8">
-      <h2>Single-node, multi-node or data centre cluster?</h2>
-      <p>These instructions cover three scenarios: single-node deployment; ideal for testing and development, multi-node deployment; suitable for the edge and IoT, and data centre cluster deployment; designed for large and complex architectures.</p>
+      <h2>Single-node, multi-node or large scale?</h2>
+      <p>Choose the OpenStack installation option that suits you best:</p>
     </div>
   </div>
   <div class="row u-equal-height">
     <div class="col-4 p-card">
-      <h3>Single&dash;node deployment</h3>
+      <h3 class="p-card__title">Single&dash;node deployment</h3>
+      <p class="p-card__content">Suitable for testing and development use cases:</p>
       <ul class="p-list">
-        <li class="p-list__item is-ticked">Uses MicroStack &ndash; snap&dash;based installation</li>
-        <li class="p-list__item is-ticked">Single machine needed</li>
-        <li class="p-list__item is-ticked">16GB RAM or more</li>
-        <li class="p-list__item is-ticked">Multi-core processor</li>
-        <li class="p-list__item is-ticked">50GB of free disk space</li>
-        <li class="p-list__item is-ticked">Ubuntu 18.04 LTS or later</li>
-        <li class="p-list__item is-ticked">Includes all key OpenStack components</li>
-        <li class="p-list__item is-ticked">Fully compatible with the upstream</li>
-        <li class="p-list__item is-ticked">Supports development iteration</li>
+        <li class="p-list__item is-ticked">One physical machine needed</li>
+        <li class="p-list__item is-ticked">Uses MicroStack &ndash; OpenStack in a snap</li>
+        <li class="p-list__item is-ticked">Core services included</li>
+        <li class="p-list__item is-ticked">All services on a single node</li>
+        <li class="p-list__item is-ticked">Opinionated OpenStack</li>
+        <li class="p-list__item is-ticked">Straightforward installation</li>
+        <li class="p-list__item is-ticked">OpenStack “on-rails</li>
```

Comment:
```suggestion
        <li class="p-list__item is-ticked">OpenStack "on-rails"</li>
```
Close quotation marks and use vertical quotation marks